Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Identity_Analytics_README.md

```diff
@@ -0,0 +1,80 @@
+The `Cortex XDR - Identity Analytics` playbook is designed to handle Cortex XDR Identity Analytics alerts and executes the following:
+
+Analysis:
+- Enriches the IP and the account, providing additional context and information about these indicators.
+
+Verdict:
+- Determines the appropriate verdict based on the data collected from the enrichment phase.
+
+Investigation:
+- Checks for related XDR alerts to the user by Mitre tactics to identify malicious activity.
+- Checks for specific arguments for malicious usage from Okta using the 'Okta User Investigation' sub-playbook.
+- Checks for specific arguments for malicious usage from Azure using the 'Azure User Investigation' sub-playbook.
+
+Verdict Handling:
+- Handles malicious alerts by initiating appropriate response actions, including blocking malicious IP and revoking or clearing user's sessions.
+- Handles non-malicious alerts identified during the investigation.
+
+The playbook is used as a sub-playbook in ‘Cortex XDR Alerts Handling v2’.
+
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+* Azure - User Investigation
+* Cloud Credentials Rotation - Azure
+* Okta - User Investigation
+* Cortex XDR - Get entity alerts by MITRE tactics
+* Block IP - Generic v3
+* Cloud IAM Enrichment - Generic
+* Account Enrichment - Generic v2.1
+
+### Integrations
+
+* XDR_iocs
+* CortexXDRIR
+* XQLQueryingEngine
+
+### Scripts
+
+* SetAndHandleEmpty
+
+### Commands
+
+* okta-clear-user-sessions
+* xdr-get-cloud-original-alerts
+* setIncident
+* ip
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| AlertName | Alert Name. |  | Optional |
+| alert_id | Alert ID. |  | Optional |
+| IPAddress | IP Address from the XDR Alert. |  | Optional |
+| Username | User name. |  | Optional |
+| RelatedAlertsThreshold | This is the minimum threshold for XDR related alerts, based on MITRE tactics used to identify malicious activity by the user in the last 1 day.<br/> | 5 | Optional |
+| FailedLogonThreshold | This is the minimum threshold for user login failures within the last 1 day.<br/>example: If this input is set to '30', and the 'Okta - User Investigation' or the 'Azure - User Investigation' sub-playbooks have found 31 failed login attempts - It will classify this behavior as malicious activity.<br/>The default value is '30'. | 30 | Optional |
+| OktaSuspiciousActivitiesThreshold | This is the minimum threshold for suspicious Okta activity events by the user in the last 1 day.<br/>example: If this input is set to '5', and the 'Okta - User Investigation' sub-playbooks have found 6 events of suspicious activity by the user - It will classify this behavior as malicious activity.<br/>The default value is '5'. | 5 | Optional |
```

Comment:
```suggestion
| FailedLogonThreshold | This is the minimum threshold for user login failures within the last 1 day.<br/>For example: If this input is set to '30', and the 'Okta - User Investigation' or the 'Azure - User Investigation' sub-playbooks have found 31 failed login attempts - It will classify this behavior as malicious activity.<br/>The default value is '30'. | 30 | Optional |
| OktaSuspiciousActivitiesThreshold | This is the minimum threshold for suspicious Okta activity events by the user in the last 1 day.<br/>For example: If this input is set to '5', and the 'Okta - User Investigation' sub-playbooks have found 6 events of suspicious activity by the user - It will classify this behavior as malicious activity.<br/>The default value is '5'. | 5 | Optional |
```